src/pineko/configs.py

```diff
@@ -11,6 +11,18 @@
 configs = {}
 "Holds loaded configurations"
 
+needed_keys = [
+    "ymldb",
+    "operator_cards",
+    "grids",
+    "operator_card_template_name",
+    "theory_cards",
+    "fktables",
+    "ekos",
+]
+
+needed_files = ["operator_card_template_name"]
```

Comment:
same as above:
```suggestion
NEEDED_FILES = ["operator_card_template_name"]
```
(Python has little to no actual types, so it relies on conventions)